docs/editor_manual/copying_aliasing_existing_pages.md

```diff
@@ -35,18 +33,12 @@ Here is how to create an alias to an existing page:
 ![Clicking the change button during the copy page form in order to change the parent of the copied page.](../_static/images/screen12.7_1_alias_choose_parent_page_button.png)
 
 -   Select and click on the desired parent page.
-
-![Selecting and clicking the desired parent page.](../_static/images/screen12.7_2_alias_choose_new_parent_page.png)
-
 -   Make sure the _Alias_ checkbox is ticked.
 -   You can then complete the page aliasing by clicking on the _Copy this page_ button.
 
 ![Clicking on the Copy this page button to confirm aliasing.](../_static/images/screen12.7_3_alias_confirm_changes.png)
 
 -   The aliased page will then appear under the new parent in the Explorer.
-
-![The aliased page appearing under the new parent page in the Explorer page.](../_static/images/screen12.7_3_alias_page_new_parent.png)
```

Comment:
Removed because this was very painful to set up (one-off screenshot taking of a success message on a listing requires deleting any precedent attempts beforehand). This didn’t feel useful enough of a visual to warrant the effort.